[cleanup] Remove InsertQuantDeQuantNode

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #25217 [quant][graphmode] Integration tests for qconfig_dict
* #25127 [quant][graphmode] insert_quant_dequant work with qconfig_dict
* #25069 [graphmode] insert_observers use qconfig_dict
* **#25000 [cleanup] Remove InsertQuantDeQuantNode**
* #24999 [cleanup] Remove insert_observers pass
* #24998 [cleanup] Remove deprecated graph mode quantization tests
* #25209 [fix] skip tests if fbgemm is not supported for test_quantizer.py

Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17001139](https://our.internmc.facebook.com/intern/diff/D17001139)

jerryzh168 committed:
[cleanup] Remove InsertQuantDeQuantNode

Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

ZolotukhinM reviewed (approved):
Looks good assuming everything still works :)

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17001139](https://our.internmc.facebook.com/intern/diff/D17001139)

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17001139](https://our.internmc.facebook.com/intern/diff/D17001139)

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17001139](https://our.internmc.facebook.com/intern/diff/D17001139)

jerryzh168 committed:
Update on "[cleanup] Remove InsertQuantDeQuantNode"


Summary:
Remove deprecated insert_quantdequant pass

Test Plan:
.

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17001139](https://our.internmc.facebook.com/intern/diff/D17001139)
